lambda

hahawang1986 opened the pull request:
two different way to automatically load data from sourced s3 to destinated s3

hahawang1986 committed:
lambda

two different way to automatically load data from sourced s3 to destinated s3

RuixuanDSBA commented on data pipeline code source/event trigger lambda.ipynb:
```diff
@@ -0,0 +1,70 @@
+{
```
can we directly run it on lambda?

RuixuanDSBA commented on data pipeline code source/schdule trigger lambda.ipynb:
```diff
@@ -0,0 +1,83 @@
+{
```
Is this coded triggered every day?

RuixuanDSBA reviewed (approved):
Good to have new code

hahawang1986 committed:
Add files via upload

use glue to extract features from raw dataset and form the feature table for the input of Segamaker

hahawang1986 committed:
Delete glue-ETL-SQL.ipynb

hahawang1986 committed:
Add files via upload

extract features from raw dataset using glue, the feature table can be used as input scource for ML(Sagemaker).

RuixuanDSBA reviewed (approved):
updata glue-etl-sql
